Tests/Selenium2TestCase/LogTest.php

```diff
@@ -52,73 +52,74 @@
  */
 class Tests_Selenium2TestCase_LogTest extends Tests_Selenium2TestCase_BaseTestCase
 {
-	private static $expectedLogTypes = array(
-		'default' => array(
-			'browser'
-		),
-		'firefox' => array(
-			'browser',
-			'driver',
-			'client',
-			'server'
-		),
-		'chrome' => array(
-			'browser',
-			'client',
-			'server'
-		)
-	);
+    private static $expectedLogTypes = array(
+        'default' => array(
+            'browser'
+        ),
+        'firefox' => array(
+            'browser',
+            'driver',
+            'client',
+            'server'
+        ),
+        'chrome' => array(
+            'browser',
+            'client',
+            'server'
+        )
+    );
 
-	private static $jsInlineErrors = array(
-		'default' => '',
-		'chrome' => 'html/test_log.html 4 Uncaught TypeError: Cannot read property \'inlineError\' of null',
-		'firefox' => 'TypeError: null has no properties'
-	);
+    private static $jsInlineErrors = array(
+        'default' => '',
+        'chrome' => 'html/test_log.html 4 Uncaught TypeError: Cannot read property \'inlineError\' of null',
+        'firefox' => 'TypeError: null has no properties'
+    );
 
     public function testLogType()
     {
-	    if($this->getBrowser() == 'opera')
-		    $this->markTestSkipped('Opera is very unstable support this command');
+        $this->markTestSkipped('Unsupported command');
```

Comment:
I think they still might work with Chrome and IE but this command is not in the new [WebDriver specifications](https://w3c.github.io/webdriver/webdriver-spec.html) and will not be implemented by the gecko driver.